backend/app/api/endpoints/memory.py

```
@@ -0,0 +1,256 @@
+
+            if memory_id and content:
+                memories.append(MemoryResponse(
+                    id=str(memory_id),
+                    content=content,
+                    created_at=str(created_at) if created_at else None,
+                    updated_at=str(updated_at) if updated_at else None,
+                ))
+
+        return MemoryListResponse(memories=memories, total=len(memories))
+
+    except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)